subprojects/core-api/src/main/java/org/gradle/api/artifacts/ProjectDependency.java

```diff
@@ -28,9 +29,38 @@
 @HasInternalProtocol
 @SuppressWarnings("deprecation") // Because of SelfResolvingDependency
 public interface ProjectDependency extends ModuleDependency, SelfResolvingDependency {
+
+    /**
+     * Get the path to the project that this dependency refers to relative to its owning build.
+     *
+     * @see Project#getPath()
+     *
+     * @since 8.11
+     */
+    String getPath();
```

Comment:
Do we also want a `getName`? 

I could see it maybe being useful for reporting in some cases, though the name is not really useful otherwise

Reply:
Not at the moment